src/frontend/sass/components/hyperlinks.theme.scss

```diff
@@ -0,0 +1,9 @@
+// Hyperlinks
+@mixin app-hyperlinks($theme, $app-theme) {
+
+  $accent: map-get($theme, accent);
+
+  a {
```

Comment:
We've lost the visited colour. Not sure if this is important though

Reply:
I did look at angular material design:

https://github.com/angular/material/commit/81b44a47cdf6c23f6811ebbb4ccaad17fd0b81af

We could add md-accent to all of our links to get them styled. I will look at that - they don't specify a visited state either.